tests/config/invalid_config.yaml

```diff
@@ -0,0 +1,40 @@
+---
+llm_providers:
+  - name: p1
+    type: bam_invalid
+    url: "https://url1"
+    credentials_path: tests/config/secret/apitoken
+    models:
+      - name: m1
+        url: "https://murl1"
+        credentials_path: tests/config/secret/apitoken
+        context_window_size: 450
+        parameters:
+          max_tokens_for_response: 100
+      - name: m2
+        url: "https://murl2"
+  - name: p2
+    type: openai
+    url: "https://url2"
+    models:
+      - name: m1
+        url: "https://murl1"
+      - name: m2
+        url: "https://murl2"
+ols_config:
+  reference_content:
+    product_docs_index_path: "tests/config"
+    product_docs_index_id: product
+  conversation_cache:
+    type: memory
+    memory:
+      max_entries: 1000
+  logging_config:
+    logging_level: INFO
+  default_provider: p1
+  default_model: m1
+  user_data_collection:
+    transcripts_disabled: true
+dev_config:
+  disable_auth: true
+  disable_tls: true
```

Comment:
Should it be here? What is invalid?

Reply:
it is required by some benchmark. And `type` is invalid?

Reply:
so can you make a note on the line, to be visible on first sight why it is invalid?

Reply:
done @onmete